### PR TITLE
added changelog update records for 1.18.0, 1.17.7, 1.16.11, and 1.15.16

### PR DESCRIPTION
**Pull request by lursu**
### Description
What does this PR do?

### TODO only if you're a HashiCorp employee
- [ ] **Backport Labels:** If this PR is in the ENT repo and needs to be backported, backport  
  to N, N-1, and N-2, using the `backport/ent/x.x.x+ent` labels. If this PR is in the CE repo, you should only backport to N, using the `backport/x.x.x` label, not the enterprise labels.
    - [ ] If this fixes a critical security vulnerability or [severity 1](https://www.hashicorp.com/customer-success/enterprise-support) bug, it will also need to be backported to the current [LTS versions](https://developer.hashicorp.com/vault/docs/enterprise/lts#why-is-there-a-risk-to-updating-to-a-non-lts-vault-enterprise-version) of Vault. To ensure this, use **all** available enterprise labels.
- [ ] **ENT Breakage:** If this PR either 1) removes a public function OR 2) changes the signature
  of a public function, even if that change is in a CE file, _double check_ that
  applying the patch for this PR to the ENT repo and running tests doesn't
  break any tests. Sometimes ENT only tests rely on public functions in CE
  files.
- [ ] **Jira:** If this change has an associated Jira, it's referenced either
  in the PR description, commit message, or branch name.
- [ ] **RFC:** If this change has an associated RFC, please link it in the description.
- [ ] **ENT PR:** If this change has an associated ENT PR, please link it in the
  description. Also, make sure the changelog is in this PR, _not_ in your ENT PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
-## 1.18.0-rc1
-## September 18, 2024
+## 1.18.0 Enterprise
```
Comment (tomcf-hcp) [inline CHANGELOG.md:5]: Should this have `Enterprise`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -63,7 +64,7 @@ engine. [[GH-27620](https://github.com/hashicorp/vault/pull/27620)]
 * **Adaptive Overload Protection (enterprise)**: Enables Adaptive Overload Protection
 for write requests as a GA feature (enabled by default) for Integrated Storage.
 * **Audit Entry Exclusion**: Audit devices support excluding fields from entries being written to them, with expression-based rules (powered by go-bexpr) to determine when the specific fields are excluded.
-* **Workload Identity Federation UI for AWS (enterprise)**: Add WIF fields to AWS secrets engine. [[GH-28148](https://github.com/hashicorp/vault/pull/28148)]
+* **Feature Name**: Add WIF fields to AWS secrets engine. [[GH-28148](https://github.com/hashicorp/vault/pull/28148)]
```
Comment (digivava) [inline CHANGELOG.md:67]: What happened with this one I wonder? The original line looks more correct.

Comment (lursu) [reply CHANGELOG.md:67]: not sure? but can manually revert if we think that's the right thing to do.

File: CHANGELOG.md (hunk 1 above)
Comment (lursu) [reply CHANGELOG.md:5]: it showed it in the example from the docs[here](https://hashicorp.atlassian.net/wiki/spaces/VAULT/pages/3303833604/1.18.0+GA+1.17.7+1.16.11+1.15.16#Creating-the-Changelog-PR-on-main), however when I look at 1.17.0 for example it does not have that so I'll remove.

File: CHANGELOG.md (hunk 2 above)
Comment (digivava) [reply CHANGELOG.md:67]: yeah not sure how it happened but agreed, I think just manually replacing that with the original text would be best

File: CHANGELOG.md [hunk 3]
```diff
@@ -551,6 +581,23 @@ autopilot to fail to discover new server versions and so not trigger an upgrade.
 * ui: fixed a bug where the replication pages did not update display when navigating between DR and performance [[GH-26325](https://github.com/hashicorp/vault/pull/26325)]
 * ui: fixes undefined start time in filename for downloaded client count attribution csv [[GH-26485](https://github.com/hashicorp/vault/pull/26485)]
 
+## 1.16.11
```
Comment (digivava) [inline CHANGELOG.md:584]: This is an enterprise-only release, right? If so (at least according to our instructions) I think this needs to have Enterprise after it. Same with the 1.15.16 one.

Comment (lursu) [reply CHANGELOG.md:584]: AH that is where it is needed ty. is the 1.17.7 on an ent only as well?

Comment (digivava) [reply CHANGELOG.md]: Oh I think you're right yes, because we only release N for CE now, not even N-1.

Comment (lursu) [reply CHANGELOG.md]: done

Comment (digivava) [reply CHANGELOG.md]: Looking here yes, only the enterprise binary is available: https://releases.hashicorp.com/vault/

**Review by digivava — APPROVED** [review]
Seems right to me! Thanks for figuring this out!

**Review by tomcf-hcp — APPROVED** [review]

**Pull request merged**